api/store/mongo/migrations/migration_23.go

```diff
@@ -0,0 +1,39 @@
+package migrations
+
+import (
+	"context"
+	"fmt"
+
+	//"github.com/shellhub-io/shellhub/pkg/models"
+	"github.com/sirupsen/logrus"
+	migrate "github.com/xakep666/mongo-migrate"
+	"go.mongodb.org/mongo-driver/bson"
+	//"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+var migration_23 = migrate.Migration{
+	Version:     23,
+	Description: "change dot in namespace name and hostname to -",
+	Up: func(db *mongo.Database) error {
+		logrus.Info("Applying migration 23 - Up")
+		result, err := db.Collection("namespaces").UpdateMany(context.TODO(), bson.D{}, []bson.M{
+			{
+
+				"$set": bson.M{"name": bson.M{
+					"$replaceOne": bson.M{"input": "$name", "find": ".", "replacement": "-"},
```

Comment:
What happens if there are multiple `.` on the field?

Reply:
I changed to "replaceAll" and now works with multiple dots